spec/example_app/app/views/admin/customers/_collection_item_actions.html.erb

```diff
@@ -12,7 +12,7 @@
     [namespace, resource],
     class: "text-color-red",
     method: :delete,
-    data: { confirm: t("administrate.actions.confirm") }
+    data: { turbo_confirm: t("administrate.actions.confirm") }
```

Comment:
I think the specs are failing because the `method: :delete` bit should also be changed to a `turbo_method: :delete` data item.